<launch>
     <arg name="lat" default="35.518404"/>
     <arg name="lon" default="-97.766172"/>

     <node pkg="mavros" type="initialize_ekf.py" respawn="false" name="ekf_start" output="screen" clear_params="true">
          <param name="fallback_latitude"       type="double" value="$(arg lat)" />
          <param name="fallback_longitude"      type="double" value="$(arg lon)"/>
     </node>
</launch>
